src/lib/logical_query_plan/data_dependencies/functional_dependency.cpp

```diff
@@ -6,7 +6,11 @@ namespace hyrise {
 
 FunctionalDependency::FunctionalDependency(ExpressionUnorderedSet init_determinants,
                                            ExpressionUnorderedSet init_dependents)
-    : determinants(std::move(init_determinants)), dependents(std::move(init_dependents)) {
+    : FunctionalDependency(std::move(init_determinants), std::move(init_dependents), true) {}
```

Comment:
> I've recently seen this in somebody else's code and I thought it improves readability. Maybe you find it useful as well?

While they improve readability, I personally dislike adding comments like this because they need to be maintained as well and can easily become out of sync. Also, in this case, the constructor that is called is just two lines below that, so one can easily look there to see how the parameter is named.

> I also wondered why this is being set to true. Maybe it makes sense to explain this in a comment?

I added a comment for that